### PR TITLE
docs(operatorhub): Info about auto-releasing the bundle.

### PR DESCRIPTION
**Pull request by SteveMattar**
Depends on https://github.com/submariner-io/releases/pull/272

Signed-off-by: Steve Mattar <smattar@redhat.com>

### CODE REVIEW

File: src/content/development/release-process/_index.en.md [hunk 1]
```diff
@@ -61,6 +62,12 @@ must also be created.
 It's best to start working with the broader community to create release notes well before the release. Create a PR to start the process, and
 work with contributors to get everything added and reviewed.
 
+### Bundle (Final Releases)
+
+If you're creating a release meant for general consumption, not a milestone or release candidate you may also want to publish the bundle
```
Comment (nyechiel) [inline src/content/development/release-process/_index.en.md]:
```suggestion
If you're creating a release meant for general consumption, i.e not an internal milestone release or a release candidate, you may want to publish the bundle
```

File: src/content/development/release-process/_index.en.md [hunk 2]
```diff
@@ -92,7 +99,7 @@ You can run the process without pushing the PR automatically (obviating the need
 To do so, run the `make` command with `dryrun=true`.
 {{% /notice %}}
 
-The command runs, gathers the data for the release, updates the release YAML and pushes it for review. Once the review process is done,
+The command runs, gathers the data for the release and updates the release YAML and pushes it for review. Once the review process is done,
```
Comment (nyechiel) [inline src/content/development/release-process/_index.en.md]: I am not sure why this change is required? The existing text looks good to me.

File: src/content/development/release-process/_index.en.md [hunk 3]
```diff
@@ -122,7 +129,23 @@ The release automation process can create stable branches for you. To do so, nav
    status: branch
    ```
 
-3) Commit your changes, create a pull request, and have it reviewed.
+3) For final releases please add the bundle fields.
```
Comment (nyechiel) [inline src/content/development/release-process/_index.en.md]:
```suggestion
3) For final releases please add the bundle fields as follows:
```

**Comment by nyechiel** [comment]
Thanks @SteveMattar - I've added a few comments. Please also note that the Markdown needs to be fixed.

File: src/content/development/release-process/_index.en.md (hunk 2 above)
Comment (SteveMattar) [reply src/content/development/release-process/_index.en.md]: It's a mistake my IDE has auto-correction enabled. Thanks :+1:

**Review by dfarrell07 — APPROVED** [review]

File: src/content/development/release-process/_index.en.md [hunk 4]
```diff
@@ -61,6 +62,12 @@ must also be created.
 It's best to start working with the broader community to create release notes well before the release. Create a PR to start the process, and
 work with contributors to get everything added and reviewed.
 
+### Bundle (Final Releases)
+
+If you're creating a release meant for general consumption, i.e. not an internal milestone release or a release candidate, you may want
+to publish the bundle to OperatorHub.io. The release process will create a Pull Request against the
```
Comment (dfarrell07) [inline src/content/development/release-process/_index.en.md:68]: I guess this depends on https://github.com/submariner-io/releases/pull/272.

**Review by nyechiel — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Comment by dfarrell07** [comment]
As discussed on the PR/Issue scrub today, there are a number of other large tasks that this depends on (subctl split -> Operator SDK upgrades -> fixing the bundle -> releasing the bundle). Closing for now.
